3.9/graphs-general-graphs-management.md

```diff
@@ -343,10 +343,12 @@ You can still add, delete or modify edge definitions and vertex collections.
 
 Add another edge definition to the graph:
 
-`graph._extendEdgeDefinitions(edgeDefinition)`
+`graph._extendEdgeDefinitions(edgeDefinition, options)`
 
 - `edgeDefinition` (object):
   The relation definition to extend the graph
+- `options` (object):
+  Additional options related to the edgeDefinition itself.
```

Comment:
Can we add a link to the Options defined later, stating what values are allowed here?
If not I think we need to copy the "satellite" sentence to here as well, otherwise it is not clear what to do from a user-perspective. Searching for "options" is kind of hopeless.

Reply:
added.